server/lib/add_user_database.js

```diff
@@ -0,0 +1,44 @@
+const request = require('request');
+const pgClient = require('./pg_client');
+
+const getGeoLocation = (postcode, callback) => {
+  const base = 'https://api.postcodes.io/postcodes/';
+  request(base + postcode, (err, response, body) => {
+    if (err) callback(err);
+    if (!err && response.statusCode === 200) {
+      callback(err, JSON.parse(body).result);
+    }
+  });
+};
+
+const addUserDatabase = (connection, user, callback) => {
+  const name = user.name;
```

Comment:
Instead you could do lines 15-19 as:
const {name, email, facebookId, profileImgUrl, postcode, client} = user;